### DISCUSSION TITLE
What is the BeatmapSetID of a map that doesnt contain BeatmapSetID

### DISCUSSION THREAD

**Discussion by leryss** [ppy/osu#13414] [Q&A] [answered]
I've downloaded an archive of osu maps from data.ppy.sh. 
When I started parsing all these maps, I've found that there are a lot without a BeatmapSetID. 
Old maps I'm guessing. Do they have a set id? If they dont have a set id can I find them on the official website with a format similar to "https://osu.ppy.sh/beatmapsets/{BeatmapSetID}#osu/{BeatmapID}" ?

**Comment by peppy** [comment, marked as answer]
You will need to query this via the API yourself.
